demos/quiz_bot/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""The bot is designed to showcase its built-in capability to create
+and use widgets with single or multiple choices.
+"""
```

Comment:
`The bot is designed to showcase  the SingleChoiceWidget and MultiChoiceWidget widgets to tackle various tasks.`